cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt

```diff
@@ -1,10 +1,16 @@
-name "Azure Instances Utilization from Log Analytics"
+name "Azure Inefficient Instance Utilization using Log Analytics"
 rs_pt_ver 20180301
 type "policy"
-short_description "Gathers instance utilization data from Azure Log Analytics and tags underutilized instances.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/instances_log_analitics_utilization) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.1"
+short_description "Gathers instance utilization data from Azure Log Analytics from the last 30 days to identify underutilized instances and provides rightsizing recommendations. Once recommendations are generated, instances can be rightsized in an automated manner or after approval. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/azure/instances_log_analitics_utilization) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+long_description ""
 severity "low"
 category "Cost"
+info(
+  version: "2.0",
+  provider: "Azure",
+  service: "Compute",
+  policy_set: ""
```

Comment:
use this value for the policy set.

Reply:
I have collected the value for policy set from the spreadsheet and updated that in the policy - Inefficient Instance Usage